### ISSUE TITLE
PolylineGeometry drops close points

### ISSUE THREAD

**Issue by alexander-entin** [AnalyticalGraphicsInc/cesium#2542]
Hard to explain. Please, run below code and try to zoom in/out.
BTW `SimplePolylineGeometry` works correctly.
````
var C = Cesium
var p = { lon: 144.777, lat: -38.338 }
var viewer = new C.Viewer('cesiumContainer')
viewer.camera.setView({ position: C.Cartesian3.fromDegrees(p.lon, p.lat, 50) })

function points(center, scale) {
    var points = []
    for (var j = -90; j <= 90; j += 5) {
        var lon = center.lon + Math.sin(j / 180 * Math.PI) * scale
        var lat = center.lat + Math.cos(j / 180 * Math.PI) * scale
        points.push(C.Cartesian3.fromDegrees(lon, lat, 1))
    }
    points.push(points[0])
    return points
}

function polyline(points) {
    var colors = []
    for (var i = 0; i < points.length; i++) colors.push(C.Color.WHITE)
    var appearance = new C.PolylineColorAppearance()
    var instance = new C.GeometryInstance({
        geometry: new C.PolylineGeometry({
            positions: points
          , colors: colors
          , width: 3
          , vertexFormat: appearance.vertexFormat
        })
    })
    return new C.Primitive({
        geometryInstances: instance
      , appearance: appearance
      , asynchronous: false
    })
}

var primitive
viewer.clock.onTick.addEventListener(function() {
    viewer.scene.primitives.remove(primitive)
    var scale = viewer.camera.positionCartographic.height / 9e5
    primitive = polyline(points(p, scale))
    viewer.scene.primitives.add(primitive)
})
````

**Comment by mramato** [comment]
There's definitely something funky going on here.  @bagnell can you take a look?

**Comment by mramato** [comment]
This was fixed with #2609.  @alexander-entin, if you are still having an issue with the latest release, let us know.
